app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
         query = ScimRails::ScimQueryParser.new(params[:filter])
 
         groups = @company.public_send(ScimRails.config.scim_groups_scope)
-                         .where("#{ScimRails.config.scim_groups_model.connection.quote_column_name(query.group_attribute)} #{query.operator} ?", query.parameter)
+                         .where("lower(#{ScimRails.config.scim_groups_model.connection.quote_column_name(query.group_attribute).gsub("\"", "")}) = ?", query.parameter.downcase)
```

Comment:
very minor, but I would better wrap downcase in the parameter method of the query service. Just because its the service's responsibility to return 'ready' to the next usage value.